app/views/AttitudeWindow.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/AttitudeWindow
```

Comment:
same thing as in the other PR. this is a view, not a window. Name the file appropriately as well